WheelOfFortune/src/engine.cpp

```
@@ -192,16 +237,21 @@ void Engine::waitForOrder()
+	int durationInSeconds;
+
+	if (generateRandData == nullptr)
+	{
+		changeState(Stage::Idle);
+		throw std::out_of_range("Invalid RandGenerator value");
+		// TODO: Log
+		// return;
+	}
 
-	int durationInSeconds = 8 + std::rand() % m_Settings.m_MaxDurationTime;
+	(this->*generateRandData)(randomAngle, durationInSeconds);
```

cppcheck (2.10-2)

[error] Uninitialized variable

(uninitvar)